db/seeds.rb

```diff
@@ -29,8 +29,10 @@
 
   puts 'Putting users in Parking department'
   parking = Department.find_by name: 'Parking'
-  parking.users << User.all
-
+  parking.users << User.all[0...((User.count/2).floor)]
```

Comment:
actually this seems to only take equal halves, there are 9 users, this method takes 4, puts them in parking users, 4 more and puts them in transit users.  Albeit I think this is fine because users are not required to have a department, leaving one without a department here is fine.